kik_unofficial/client.py

```diff
@@ -569,15 +573,17 @@ def change_email(self, old_email, new_email):
         self.logger.info(f"Changing account email to '{new_email}'")
         return self._send_xmpp_element(account.ChangeEmailRequest(self.password, old_email, new_email))
 
-    def disconnect(self):
+    def disconnect(self, close=False):
```

Comment:
Since the purpose of this function is to close the connection the meaning of the argument `close` is not really clear